### PR TITLE
Updated socket-shop demo create & cleanup

### PR DESCRIPTION
**Pull request by MatejB**
Microservice demo was changed and now creates two namespaces socket-shop and zipkin.
Updated sample commands to reflect that change.

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.7 Features: set Milestone to `1.7` and Base Branch to `release-1.7`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box below to allow 
> reviewers fix problems on your patch and speed up the review process.
> Please delete this note before submitting the pull request.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4100)

### CODE REVIEW

**Comment by MatejB** [comment]
I signed CLA.

**Comment by luxas** [comment]
I'm not that familiar with the sock-shop, reassigning to the authors of that demo

**Comment by steveperry-53** [comment]
@errordeveloper, @lukemarsden, Will you be able to take a look at this anytime soon? If not, can you suggest another tech reviewer? Thanks.

**Comment by chenopis** [comment]
@jbeda Can you take quick look and to a tech review? thx

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -331,8 +331,7 @@ README](https://github.com/microservices-demo/microservices-demo).
 Note that the Sock Shop demo only works on `amd64`.
 
 ``` bash
-kubectl create namespace sock-shop
-kubectl apply -n sock-shop -f "https://github.com/microservices-demo/microservices-demo/blob/master/deploy/kubernetes/complete-demo.yaml?raw=true"
+kubectl apply -f "https://github.com/microservices-demo/microservices-demo/blob/master/deploy/kubernetes/complete-demo.yaml?raw=true"
```
Comment (jbeda) [inline docs/setup/independent/create-cluster-kubeadm.md]: Looking at that yaml, it looks like it doesn't actually create the namespace.  Do we perhaps need the starting `create namespace` command still?

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -351,8 +350,8 @@ front-end   10.110.250.153   <nodes>       80:30001/TCP   59s
 ```
 
 It takes several minutes to download and start all the containers, watch the
-output of `kubectl get pods -n sock-shop` to see when they're all up and
-running.
+output of `kubectl get pods -n sock-shop && kubectl get pods -n zipkin` to see 
+when they're all up and running.
```
Comment (jbeda) [inline docs/setup/independent/create-cluster-kubeadm.md]: Where and how does zipkin get installed?  I don't see that in the previous yaml.  And I don't see where the namespace gets created...

**Comment by jbeda** [comment]
Sorry for the delay in review.  If you need me to take another look feel free to poke me on slack.

**Comment by MatejB** [comment]
It seems that in the meantime yaml changed with this commit: https://github.com/microservices-demo/microservices-demo/commit/15029c9adf11d0952ffdc78c568346950172b12c#diff-7a608398a77b27e4f980d5cd06a6eedf 

Zapkine was removed and yaml does not create namespaces by itself.
I will close this issue as it was fixed by changes in microservice-demo :-)

**Pull request closed without merging**